### PR TITLE
Add plugins.md with a list of links to known community plugins

### PR DESCRIPTION
**Pull request by richbeales**
### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: PLUGINS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+⚠️💀 WARNING 💀⚠️: Review the code of any plugin you use, this allows for any Python to be executed and do malicious things. Like stealing your API keys.
+
+# Plugins List
+
+The following community plugins are known to the maintainers of Auto-GPT.  Their presence here is not an endorsement of their safety or functionality.
+
+## Prompt Plugins
+
+System Info - Inject the Operating System into the prompt
```
Comment (hdkiller) [inline PLUGINS.md]: Injects information about the Operation System into the prompt

File: PLUGINS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# Plugins List
+
+The following community plugins are known to the maintainers of Auto-GPT.  Their presence here is not an endorsement of their safety or functionality.
+
+## Prompt Plugins
+
+System Info -
+Injects information about the Operation System into the prompt
+
+https://github.com/hdkiller/Auto-GPT-SystemInfo
```
Comment (ntindle) [inline PLUGINS.md]:
```suggestion
[System Info](https://github.com/hdkiller/Auto-GPT-SystemInfo) - 
https://github.com/hdkiller/Auto-GPT-SystemInfo
```

**Review by ntindle — APPROVED** [review]
May want to link to plugins.md from the readme

**Comment by ntindle** [comment]
Closing in favor of #2883

**Pull request closed without merging**